homeWork_03/ex3_Circle/Victoria_Kacer/circle.java

```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+public class Circle {
+
+    public double radius;
+    public double pi;
+    public double calculateArea() {
+        return this.pi * this.radius * this.radius;
```

Comment:
"this." can be deleted as Circle is not extended, which in turn means that there are no fields with the same names (radius, pi)